src/Adapter/Image/ImageRetriever.php

```diff
@@ -165,47 +170,56 @@ public function getImage($object, $id_image)
         $urls = [];
         $image_types = ImageType::getImagesTypes($type, true);
         $generateHighDpiImages = (bool) Configuration::get('PS_HIGHT_DPI');
-        $ext = 'jpg';
 
-        // Get path of original uploaded image we will use to get thumbnails
+        // Get path of original uploaded image we will use to get thumbnails (original image is always .jpg)
         $originalImagePath = implode(DIRECTORY_SEPARATOR, [
             $imageFolderPath,
-            $id_image . '.' . $ext,
+            $id_image . '.jpg',
         ]);
 
+        $rewriteLink = !empty($object->link_rewrite) ? $object->link_rewrite : $object->name;
         foreach ($image_types as $image_type) {
-            // Final path thumbnail in our size
-            $thumbnailPath = implode(DIRECTORY_SEPARATOR, [
-                $imageFolderPath,
-                $id_image . '-' . $image_type['name'] . '.' . $ext,
-            ]);
+            $additionalSources = [];
+
+            if (!$this->isMultipleImageFormatFeatureActive || Configuration::get('PS_ADDITIONAL_IMAGE_JPG')) {
```

Comment:
```suggestion
            if ($this->isMultipleImageFormatFeatureActive && Configuration::get('PS_ADDITIONAL_IMAGE_JPG')) {
```
This condition doesn't seem logical, especially when looking at the other ones which make more sense and are more consistent

Reply:
Ah I think I understand now, it's the default behaviour to rely on jpg format when the feature flag is disbaled right? In that case it deserves a very clear comment here